ReText/tab.py

```diff
@@ -456,3 +470,47 @@ def openSourceFile(self, fileToOpen):
 		if exists(fileToOpen) and get_markup_for_file_name(fileToOpen, return_class=True):
 			self.p.openFileWrapper(fileToOpen)
 			return fileToOpen
+
+	def cacheRemotelyHostedImages(self, text):
+		"""
+		Search the text for remotely hosted images and cache them into /tmp/ so that
+		they don't have to be loaded over again.
+		"""
+		offset=0
+		# Look for <img/> tags
+		for match in HTML_IMG_TAG_SRC_RE.finditer(text):
+			img_src = match.groups(1)[0]
+			start_position = match.span(1)[0]
+			end_position = match.span(1)[1]
+			local_file = None
+
+			# If this <img/> tag is not remotely hosted, move on.
+			if img_src.startswith('http') is False:
+				continue
+
+			# If it is remotely hosted and we haven't seen it before,
+			# give it a unique filename and save it into /tmp/. Also, save it into
+			# a dict (seen) so that we can track it for later use and cleanup
+			if img_src not in self.seen:
+				local_file = f'/tmp/{str(uuid.uuid4())}_{img_src[img_src.rfind("/")+1:]}'
```

Comment:
`/tmp/` will not work on some platforms, such as Windows.

Please use a cross-platform solution such as [tempfile](https://docs.python.org/3/library/tempfile.html) module.

Reply:
Oh shoot, thanks! TIL'd and fixed.